src/python/data/FilenameMode.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class FilenameMode(Enum):  # {
+    EXTENSION: bool = 0
```

Comment:
var: bool = 0...

Reply:
in case of future filenamemode extension, you probably should use int

Reply:
You are right. I think typing for enums is useless at all, so, i deleted it.